packages/odyssey-react/src/components/Text/Text.tsx

```diff
@@ -71,21 +72,13 @@ export interface TextProps
    * The text color style for the text content.
    * @default body
    */
-  color?:
-    | "body"
-    | "body-inverse"
-    | "code"
-    | "danger"
-    | "danger-disabled"
-    | "sub"
-    | "heading"
```

Comment:
The desire to define a heading specific color seems useful for theming, do we have this in the heading component?